platform/macos/display_server_macos.mm

```diff
@@ -1682,6 +1682,41 @@
 	[tts stopSpeaking];
 }
 
+bool DisplayServerMacOS::is_dark_mode_supported() const {
+	if (@available(macOS 10.14, *)) {
+		return true;
+	} else {
+		return false;
+	}
+}
+
+bool DisplayServerMacOS::is_dark_mode() const {
+	if (@available(macOS 10.14, *)) {
+		if (![[NSUserDefaults standardUserDefaults] objectForKey:@"AppleInterfaceStyle"]) {
+			return false;
+		} else {
+			return ([[[NSUserDefaults standardUserDefaults] stringForKey:@"AppleInterfaceStyle"] isEqual:@"Dark"]);
+		}
+	} else {
+		return false;
+	}
+}
+
+Color DisplayServerMacOS::get_accent_color() const {
+	if (@available(macOS 10.14, *)) {
+		NSColor *color = [[NSColor controlAccentColor] colorUsingColorSpaceName:NSCalibratedRGBColorSpace];
```

Comment:
Isn't [`colorUsingColorSpaceName`](https://developer.apple.com/documentation/appkit/nscolor/1534332-colorusingcolorspacename) deprecated?

Reply:
Indeed, is seems to be deprecated, replaced with `colorUsingColorSpace:`.